packages/editor-common/web/src/Components/InfoIcon.jsx

```diff
@@ -1,14 +1,13 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import generalstyles from '../../statics/styles/general.scss';
-import Tooltip from './Tooltip';
+import Tooltip from 'wix-rich-content-common/dist/lib/Tooltip.cjs.jsx';
```

Comment:
I believe it should be like this. Then in the build it can take cjs or es depending how it's built. 
Although I'm not 100% confident about this. But let's change all of them to this. 
```suggestion
import Tooltip from 'wix-rich-content-common/dist/lib/Tooltip';
```

Reply:
done